amethyst_network/src/net_event.rs

```diff
@@ -3,38 +3,19 @@
 //! NetOwnedEvent are passed through the ECS, and contains the event's source (remote connection, usually).
 
 use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
 use uuid::Uuid;
 
 /// Network events which you can send or and receive from an endpoint.
+// TODO, Connect, connection refused, disconnect, disconnected
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub enum NetEvent<T> {
-    /// Ask to connect to the server.
-    Connect {
-        /// The client uuid.
-        client_uuid: Uuid,
-    },
-    /// Reply to the client that the connection has been accepted.
-    Connected {
-        /// The server uuid.
-        server_uuid: Uuid,
-    },
-    /// Reply to the client that the connection has been refused.
-    ConnectionRefused {
```

Comment:
Those where mostly meant for the user to handle.

Reply:
Those were not implemented and used anywhere within the crate, I see the purpose of them, but I think removing them is better than not using it.